manifests/mod/deflate.pp

```diff
@@ -12,6 +12,7 @@
     'Ratio'  => 'ratio'
   }
 ) {
+  include ::apache
   ::apache::mod { 'deflate': }
```

Comment:
i'm confused.
shouldn't that line ^ error out, and tell you to first include apache before including any modules?
see: https://github.com/puppetlabs/puppetlabs-apache/blob/master/manifests/mod.pp#L11-L13